client/ayon_ftrack/plugins/launcher_actions/show_in_ftrack.py

```diff
@@ -0,0 +1,99 @@
+"""
+Example Ftrack URL:
+
+https://pype.ftrackapp.com/#slideEntityId=38c5fec4-0aed-11ea-a454-3e41ec9bc0d6&slideEntityType=show&view=tasks&itemId=projects&entityId=38c5fec4-0aed-11ea-a454-3e41ec9bc0d6&entityType=show
+
+# This is required otherwise is url invalid view=tasks&itemId=projects&entityId=38c5fec4-0aed-11ea-a454-3e41ec9bc0d6&entityType=show
+
+- "itemId=projects" the top category (overview / projects/ reports / ...)
+    must be 'projects'
+- "view=tasks" view category 'tasks' is best
+- "entityId=38c5fec4-0aed-11ea-a454-3e41ec9bc0d6" id of entity which is in focus (project id is easiest)
+- "entityType=show" entity type of 'entityId'
+
+Entity detail in slide (on right side) can't be used on it's own: slideEntityId=38c5fec4-0aed-11ea-a454-3e41ec9bc0d6&slideEntityType=show
+- "slideEntityId=38c5fec4-0aed-11ea-a454-3e41ec9bc0d6" entity id which is showed in detail
+- "slideEntityType=show" type of 'slideEntityId' entity
+
+Warning: entityType is not entity_type!
+    entityType "show" == entity_type "Project"
+    entityType "task" == entity_type "Task", "Shot", "Library", "Folder", ...
+
+"""
+import webbrowser
+
+from ayon_core.pipeline import LauncherAction
+from ayon_core.addon import AddonsManager
+from ayon_api import get_project, get_folder_by_path
+
+
+class ShowInFtrack(LauncherAction):
+    name = "showinftrack"
+    label = "Show in Ftrack"
+    icon = "external-link-square"
```

Comment:
Shouldn't we add ftrack icon?

Reply:
If that's possible, then yes. :)

Reply:
@iLLiCiTiT if you happen to have pointers for this one? :) We'd need to add the icon to client side I think since it's not there yet, or is it?

Reply:
It is not, and they have kinda strict rules about logo.

Reply:
Added